x/retry/retry.go

```diff
@@ -163,7 +163,7 @@ func getTimeLeft(ctx context.Context, max time.Duration) (timeleft time.Duration
 func isRetryable(err error) bool {
 	// TODO(#1080) Update Error assertions to be more granular.
 	switch yarpcerrors.ErrorCode(err) {
-	case yarpcerrors.CodeInternal, yarpcerrors.CodeDeadlineExceeded:
+	case yarpcerrors.CodeInternal, yarpcerrors.CodeDeadlineExceeded, yarpcerrors.CodeUnavailable, yarpcerrors.CodeUnknown:
```

Comment:
Okay, I’ll buy Internal and Unknown, though I’m skeptical that these error classes are likely to succeed on retry. We can measure that! Perhaps rename the method `isIdempotentProcedureRetryable` to make the distinction clear.